tests/ui/nll/issue-54556-niconii.rs

```diff
@@ -1,3 +1,4 @@
+//@ check-pass
```

Comment:
Suggestion: please remark why this check-pass is added and move this directive below the top-level comment (or rather, how the test changed since it was initially added).